### PR TITLE
feat(core): add accordion component

### PR DESCRIPTION
**Pull request by saylikadam**
affects: @medly-components/core, @medly-components/theme

## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [ ] Documentation content changes
-   [ ] Application / infrastructure changes
-   [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: packages/core/src/components/Accordion/Accordion.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+<Meta title="Core" component={Accordion} parameters={{ jest: ['Accordion.test.tsx'] }} />
+
+# Accordion
+
+Accordion is a container that can either stand alone or be connected with other components.
+
+<Preview withToolbar>
+    <Story name="Accordion" parameters={{ decorators: [withKnobs] }}>
+       <SimpleAccordion />
```
Comment (gmukul01) [inline packages/core/src/components/Accordion/Accordion.stories.mdx:17]: Move the SimpleAccordion code over here only.

File: packages/core/src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { ExpandMoreIcon } from '@medly-components/icons';
+import React, { SFC, useState } from 'react';
+import Text from '../Text';
+import { Content, Wrapper, Section } from './Accordion.styled';
+import { Props } from './types';
+
+
+export const Accordion: SFC<Props> = React.memo(({ title, titleColor, bgColor, children }) => {
+    const [active, setActive] = useState(false),
+        toggleAccordion = () => setActive(active ? false : true);
```
Comment (gmukul01) [inline packages/core/src/components/Accordion/Accordion.tsx:10]: Use it like this -> toggleAccordion = useCallback(() => setActive(val => !val), []);

File: packages/core/src/components/Accordion/Accordion.stories.mdx (hunk 1 above)
Comment (saylikadam) [reply packages/core/src/components/Accordion/Accordion.stories.mdx]: done

File: packages/core/src/components/Accordion/Accordion.tsx (hunk 2 above)
Comment (saylikadam) [reply packages/core/src/components/Accordion/Accordion.tsx]: done

**Pull request merged**